src/extensions/scratch3_speech/index.js

```diff
@@ -691,23 +692,35 @@ class Scratch3SpeechBlocks {
             blocks: [
                 {
                     opcode: 'listenAndWait',
-                    text: 'Listen and Wait',
+                    text: formatMessage({
+                        id: 'speech.ListenAndWait',
+                        default: 'listen and wait',
+                        description: 'Start listening to the microphone and wait for data.'
+                    }),
                     blockType: BlockType.COMMAND
                 },
                 {
                     opcode: 'whenIHearHat',
-                    text: 'When I hear [PHRASE]',
+                    text: formatMessage({
+                        id: 'speech.whenIHear',
+                        default: 'when I hear [PHRASE]',
+                        description: 'Text on hat block that triggers when phrase is heard.'
```

Comment:
How about "Event that triggers when the text entered on the block is recognized by the speech recognition system"

Reply:
done!